packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -87,23 +89,28 @@ class DateTimeUtils {
     const newDate = momentDate.clone();
     const timeFormat = hasSeconds ? 'HH:mm:ss' : 'HH:mm';
     const date = moment(time, timeFormat, true);
+    const timeZone = momentDate.tz() || moment.tz.guess();
```

Comment:
Anywhere that does `moment.tz.guess()` can use `getLocalTimeZone`.

Reply:
changed here e3a49adabb7322a4265611a27d15cd77a0110374